### ISSUE TITLE
RSDL: more shortcuts for built-in types

### ISSUE THREAD

**Issue by ralfhandl** [oasis-open/odata-rapid#161]
`Decimal`, `TimeOfDay`, `Duration`

- [x] compiler
- [x] prose documentation
- [x] ABNF

**Comment by chrisspre** [comment]
pull request  #217  adds a test AllPrimitiveTypesGetTransformed that verifies that the following strings get parsed and transformed to the corresponding primitive type
```
"Integer":       EdmPrimitiveTypeKind.Int32
"String":        EdmPrimitiveTypeKind.String
"Boolean":       EdmPrimitiveTypeKind.Boolean
"DateTime":      EdmPrimitiveTypeKind.DateTimeOffset
"Date":          EdmPrimitiveTypeKind.Date
"Double":        EdmPrimitiveTypeKind.Double
"Decimal":       EdmPrimitiveTypeKind.Decimal
"TimeOfDay":     EdmPrimitiveTypeKind.TimeOfDay
"Duration":      EdmPrimitiveTypeKind.Duration
```

**Comment by chrisspre** [comment]
addressed by #217
